server/src/main/java/io/deephaven/server/arrow/FlightServiceGrpcImpl.java

```diff
@@ -177,7 +183,8 @@ public void doGetCustom(final Flight.Ticket request, final StreamObserver<InputS
                         msg.modColumnData = ZERO_MOD_COLUMNS; // actually no mod column data for DoGet
 
                         // translate the viewport to keyspace and make the call
-                        try (final BarrageStreamGenerator bsg = new BarrageStreamGenerator(msg)) {
+                        try (final BarrageStreamGenerator bsg = new BarrageStreamGenerator(msg, (bits, tm) -> {
```

Comment:
Same question as for `ArrowFlightUtil`: Are we just ignoring metrics for snapshots? Why not report a one-off?